{"name": "test_login[c]", "status": "broken", "statusDetails": {"message": "TypeError: 'SubRequest' object is not iterable", "trace": "request = <SubRequest 'pw_encryption' for <Function test_login[c]>>\n\n    @pytest.fixture(scope=\"function\")\n    def pw_encryption(request):\n        h1 = hashlib.md5()\n>       pw = list(request)\nE       TypeError: 'SubRequest' object is not iterable\n\nconftest.py:23: TypeError"}, "parameters": [{"name": "pw_encryption", "value": "'c'"}], "start": 1668157855880, "stop": 1668157855880, "uuid": "05e82caa-c0cc-484f-a0e5-6145034d62b9", "historyId": "ddddb4f24af0117480a81474a86b31e2", "testCaseId": "9a714c8af21ed863957756d3539d49e3", "fullName": "testcases.test_send_request.TestSendRequest#test_login", "labels": [{"name": "parentSuite", "value": "testcases"}, {"name": "suite", "value": "test_send_request"}, {"name": "subSuite", "value": "TestSendRequest"}, {"name": "host", "value": "PC-201703221222"}, {"name": "thread", "value": "15948-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testcases.test_send_request"}]}